andela/wg-wits
# [HOLD]#159687667 Enable manager to delete users and trainers from a gym

#### What does this PR do?
- Enable gym managers to delete users' and trainers' account from the gym, not the application.
- Enable gym administrators to add already existing users to the gym.
#### Description of Task to be completed?
- Modify UI to enable gym managers adding already existing users to the gym.
- Add  `GymAddExistingUserForm` form and `GymAddExistingUserView` view.
#### How should this be manually tested?
-  Clone this [branch](https://github.com/andela/wg-wits/tree/ft-more-than-one-gyms-159687667) and follow the instructions stated in [README](https://github.com/andela/wg-wits/tree/ft-allow-deleting-trainers-159236214) to set up the application.
- After setting up the application, start the server by running the command `python manage.py runserver` in the terminal with your virtual environment set.
- In the browser, log in as an Admin and go to the `gyms` page. As an admin or a gym manager, you should be able to delete, and create users as shown in the screenshots section.

#### What are the relevant pivotal tracker stories?
[159687667](https://www.pivotaltracker.com/story/show/159687667)

#### Screenshots
Screenshot before the implementation
![screen shot 2018-08-08 at 19 54 17](https://user-images.githubusercontent.com/39955305/44199677-6858bd80-a14d-11e8-8f7d-d05dc2e872e4.png)


Screenshots after implementation
<img width="1381" alt="screen shot 2018-08-16 at 12 11 26" src="https://user-images.githubusercontent.com/39955305/44199801-b7065780-a14d-11e8-8c9b-a4779e52577d.png">


<img width="1375" alt="screen shot 2018-08-16 at 12 11 45" src="https://user-images.githubusercontent.com/39955305/44199808-bb327500-a14d-11e8-8167-0f568f7112d6.png">
